compendium/DeclarativeServices/src/manager/ComponentConfigurationImpl.hpp

```diff
@@ -177,6 +177,12 @@ class ComponentConfigurationImpl
    */
   virtual std::shared_ptr<ComponentInstance> CreateAndActivateComponentInstance(const cppmicroservices::Bundle& bundle) = 0;
 
+  /**
+   * Method called to modify the configuration properties for this component configuration. Subclasses
+   * must implement this method. Returns false if the component instance has not provided a Modified method.
+   */
+  virtual bool ModifyComponentInstanceProperties() = 0;
```

Comment:
The comment should use the @return javadoc tag to describe the return values.

Reply:
Fixed